### PR TITLE
Update makefile

### PR DESCRIPTION
**Pull request by wanghaoran1988**

### CODE REVIEW

File: hack/app_sre_pr_check.sh [hunk 1]
```diff
@@ -2,31 +2,4 @@
...
-   if ! $CONTAINER_ENGINE run --rm -v $(pwd):/json quay.io/haowang/jsonschema:latest -i /json/$f /json/hack/metadata.schema.json; then
-     echo "validation failed: $f"
-     exit 1
-   else
-     echo "validation succeed"
-   fi
-done
-
-make IMAGE_REPOSITORY=${IMAGE_REPOSITORY:-app-sre} build
+make IMAGE_REPOSITORY=${IMAGE_REPOSITORY:-app-sre} build validation
```
Comment (qhua948) [inline hack/app_sre_pr_check.sh:5]: no need validation here, since build depends on validation

File: hack/schema_validation.sh [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+   else
+     echo "convert succeed"
+   fi
+done
+#Verify the metadata.json are valid
+jfiles=$(find . -name 'metadata.json')
+for f in $jfiles
+do
+   echo "validating the jsonschema for $f"
+   if ! $CONTAINER_ENGINE run --rm -v $(pwd):/json quay.io/haowang/jsonschema:latest -i /json/$f /json/hack/metadata.schema.json; then
```
Comment (feichashao) [inline hack/schema_validation.sh:24]: is `haowang` expected here?

Comment (wanghaoran1988) [reply hack/schema_validation.sh:24]: yes, it's in my own repo, we need move this somewhere finally.

File: hack/app_sre_pr_check.sh (hunk 1 above)
Comment (wanghaoran1988) [reply hack/app_sre_pr_check.sh]: @qhua948 Done

**Pull request merged**